Library/Homebrew/extend/os/diagnostic.rb

```diff
@@ -1,2 +1,3 @@
 require "diagnostic"
 require "extend/os/mac/diagnostic" if OS.mac?
+require "extend/os/linux/diagnostic" if OS.linux?
```

Comment:
```ruby
if OS.mac?
  require "extend/os/mac/diagnostic"
elsif OS.linux?
  require "extend/os/linux/diagnostic"
end
```

Reply:
But the current one takes only two lines instead of 5... Is there any reason to use the classical `if/elsif`?

Reply:
For consistency with the others. I believe it's Mike's preference.
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/os/cleaner.rb